pkg/iss/bucketgroup.go

```diff
@@ -59,7 +59,9 @@ func (buckets bucketGroup) Select(bucketIDs []int) bucketGroup {
 // Thus, the same request may map to some bucket in one group and to a different bucket in a different group,
 // even if the former bucket is part of the latter group.
 func (buckets bucketGroup) RequestBucket(reqRef *requestpb.RequestRef) *requestBucket {
-	bucketID := int(reqRef.ClientId+reqRef.ReqNo) % len(buckets) // If types change, this might need to be updated.
+	// TODO: do we need use clientID here?
```

Comment:
I guess we don't. Unless all clients submit requests in lock step, I can't really imagine it causing any trouble.

Reply:
ok